### ISSUE TITLE
GitHub Pages support

### ISSUE THREAD

**Issue by kevinSuttle** [certbot/certbot#1748]
I'm not the first one to think of this. https://github.com/kevinSuttle/github/issues/29

**Comment by swrobel** [comment]
:+1:

**Comment by nubela** [comment]
*Full disclosure: I work at Kloudsec.*

But anyways, *[Kloudsec](https://kloudsec.com) solves the problem of getting custom domained Github Pages to HTTPS*. See [Kloudsec for Github Pages](https://kloudsec.com/github-pages).

Kloudsec is a minimal CDN (open) platform. Because it is an open platform, we also implemented LetsEncrypt CA to auto-provision SSL if you enable the `One-click Encryption` plugin.

**All you have to do is to update your custom domain to point to our CDN's IP address**. (Unlike Cloudflare, you can keep your nameservers)

---

In other words, [Kloudsec for Github Pages](https://kloudsec.com/github-pages) solves the 2 biggest issues with custom domains on GIthub pages:

1. No CDN support for apex domains
2. No HTTPS for custom domains

You also get all the other plugins for free:

* Page Optimizer (Pagespeed optimisations)
* Service Doctor (Performance Analytics)

*We have a [public Telegram chat group for Kloudsec](https://telegram.me/joinchat/AShWiQeusGPmWT2NxAfC6g) if you need help*

**Comment by rebelliard** [comment]
Kloudsec is dead. https://www.reddit.com/r/webdev/comments/4s3kmf/got_an_email_saying_that_kloudsec_will_be/

**Comment by sydneyli** [comment]
This is [a thing now](https://blog.github.com/2018-05-01-github-pages-custom-domains-https/)!
